modules/bookcollection.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+import Books from './books.js';
+
+export class BookCollection {
+  constructor() {
+    this.bookObj = [];
+    this.bookList = document.querySelector('#bookList');
+    this.title = document.querySelector('#title');
+    this.arthur = document.querySelector('#arthur');
+    this.addBtn = document.querySelector('#addBtn');
+
+    if (localStorage.getItem('inputArr')) {
+      this.bookObj = JSON.parse(localStorage.getItem('inputArr'));
+      this.displayBooks();
+    }
+
+    this.addBtn.addEventListener('click', this.newBook.bind(this));
+  }
+
+  newBook() {
+    const title = this.title.value;
+    const arthur = this.arthur.value;
+    const book = new Books(title, arthur);
+    this.bookObj.push(book);
+    this.displayBooks();
+    this.storeBook();
+    this.title.value = '';
+    this.arthur.value = '';
+  }
+
+  removeBook(index) {
+    this.bookObj.splice(index, 1);
+    this.displayBooks();
+    this.storeBook();
+  }
+
+  displayBooks() {
+    this.bookList.innerHTML = '';
+    this.bookObj.forEach((book, index) => {
+      const bookItem = document.createElement('li');
+
+      const paragraph = document.createElement('p');
+
+      const span1 = document.createElement('span');
+      span1.innerHTML = book.title;
+
+      const span2 = document.createElement('span');
+      span2.innerHTML = `by ${book.arthur}`;
+
+      const removeBtn = document.createElement('button');
+      removeBtn.innerHTML = 'Remove';
```

Comment:
- [ ] You have so far reached a great milestone in achieving the project requirements 👏 .  However, as the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#project-requirements) suggest, I would advise you to use the arrow function for your methods. This is to show that you are updated in the usage of ES6 of js and it is an edge in the industry. Kindly apply the changes to all your methods ☺️ .